src/native/external/README.md

```diff
@@ -0,0 +1,72 @@
+# Native external libraries
+
+This folder contains the source code of third party native libraries that are used as dependencies for Runtime.
```

Comment:
```suggestion
This folder contains the source code of vendored third party native libraries that are used as dependencies for dotnet/runtime.
```

Reply:
Pardon my ignorance: What does vendored refer to? Why would all these libraries be considered as such?

Reply:
"Vendored in" libraries are libraries we include as source in our repository. https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Strategy-For-External-Source.md has the definition.

Other dependencies (like llvm-project), we maintain as a fork. Others we depend on system packages (openssl, our compiler toolchain). In the future, we may download some (test-only) dependencies that aren't needed in source-build scenarios and would only bloat the repo. This README only applies to vendored dependencies.